packages/terra-application-navigation/src/tabs/_TabCount.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useRef, useLayoutEffect } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+
+import styles from './TabCount.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The child tabs to be placed in the menu.
+   */
+  value: PropTypes.number,
+  /**
+   * Whether or not the count represents rolled up counts.
+   */
+  isRollup: PropTypes.bool,
+  /**
+   * @private
+   * Object containing intl APIs.
+   */
+  intl: intlShape,
+};
+
+const TabCount = ({ value, isRollup, intl }) => {
+  const countRef = useRef();
+  const previousValueRef = useRef(value);
+
+  function handleAnimation() {
+    if (countRef.current) {
+      countRef.current.setAttribute('data-count-pulse', 'false');
+      countRef.current.removeEventListener('animationend', handleAnimation);
+    }
+  }
+
+  useLayoutEffect(() => {
```

Comment:
I've seen this code in a couple of places across files, have you considered making this a custom hook and providing it the `utils/helpers.js`. It would dry things up some.

Reply:
Good idea. Updated here: 166b049d043ca768424c81b7bae24b1ede86ded7

Reply:
![Noice](https://media.giphy.com/media/4LqvtG82IrSXS/giphy.gif)